quick_start/kusto_sample_app.py

```diff
@@ -0,0 +1,261 @@
+# Todo - Start Here:
+#  1) Run setup.bat to install necessary dependencies.
+#  2) Follow the To-Do comments for instructions, tips and reference material
+#  3) run the script
+
+# Todo (Yochai) cleanup the import section when done
+import io
+import os
+import time
+import typing
+
+from datetime import timedelta
+
+from azure.kusto.data.exceptions import KustoClientError, KustoServiceError
+from azure.kusto.data.helpers import dataframe_from_result_table
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder, ClientRequestProperties
+
+from azure.kusto.ingest import (
+    QueuedIngestClient,
+    IngestionProperties,
+    FileDescriptor,
+    BlobDescriptor,
+    StreamDescriptor,
+    DataFormat,
+    ReportLevel,
+    IngestionMappingType,
+    KustoStreamingIngestClient,
+)
+
+# Todo - Config (Auto-Filled properties):
+kustoUri = "https://yogiladadx.westeurope.dev.kusto.windows.net"

```

Comment:
if you want, we can read user input here

Reply:
The idea is for this to include place holders which will later be filled automatically by one click.
I would rather limit the amount of user duct-tape code, so that this sample would include mostly code which can be quickly adapted for the dev needs.